docs/using-the-aws-driver/OktaAuthentication.md

```diff
@@ -0,0 +1,51 @@
+| `IDP_ENDPOINT`                |   Yes    | The hosting URL for the Okta service that you are using to authenticate into AWS Aurora.                                                                                                                       | `null`                   | `ec2amaz-ab3cdef.example.com`                          |
+| `APP_ID`                      |   Yes    | The Amazon Web Services (AWS) app [configured](https://help.okta.com/en-us/content/topics/deploymentguides/aws/aws-configure-aws-app.htm) on Okta.                                                             | `null`                   | `ec2amaz-ab3cdef.example.com`                          |
+| `IAM_ROLE_ARN`                |   Yes    | The ARN of the IAM Role that is to be assumed to access AWS Aurora.                                                                                                                                            | `null`                   | `arn:aws:iam::123456789012:role/adfs_example_iam_role` |
+| `IAM_IDP_ARN`                 |   Yes    | The ARN of the Identity Provider.                                                                                                                                                                              | `null`                   | `arn:aws:iam::123456789012:saml-provider/adfs_example` |
+| `AWS_REGION`                  |   Yes    | The AWS region where the identity provider is located.                                                                                                                                                         | `null`                   | `us-east-2`                                            |
+| `USERNAME`                    |   Yes    | The Username must be set to the [IAM database user](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html).                                                                       | `null`                   | `us-east-2`                                            |
+| `IDP_PORT`                    |    No    | The port that the host for the authentication service listens at.                                                                                                                                              | `urn:amazon:webservices` | `urn:amazon:webservices`                               |
+| `IAM_HOST`                    |    NO    | Overrides the host used to generate the authentication token. This is useful when you are connecting using a custom endpoint, since authentication tokens need to be generated using the RDS/Aurora endpoints. | `NULL`                   | `database.cluster-hash.region.rds.amazonaws.com`       |
```

Comment:
The one yelling NO
```suggestion
| `IAM_HOST`                    |    No    | Overrides the host used to generate the authentication token. This is useful when you are connecting using a custom endpoint, since authentication tokens need to be generated using the RDS/Aurora endpoints. | `NULL`                   | `database.cluster-hash.region.rds.amazonaws.com`       |
```